extensions/amp-form/0.1/amp-form.js

```diff
@@ -234,9 +234,15 @@ export class AmpForm {
         continue;
       }
 
-      if (input.type == 'checkbox' || input.type == 'radio') {
-        data[input.name] = input.checked ? input.value : '';
-      } else {
+
+      if (input.type != 'checkbox' && input.type != 'radio') {
+        data[input.name] = input.value;
+      } else if (input.type == 'checkbox' && input.checked) {
+        if (data[input.name] === undefined) {
+          data[input.name] = [];
+        }
+        data[input.name].push(input.value);
```

Comment:
cc/ @dvoytenko just to make sure you're good with this? This makes sense to me, but wanna make sure because HTML5 recommendation might be against it - browsers seem to still support it.